showcase/src/app/demo/header/header.component.ts

```diff
@@ -0,0 +1,81 @@
+import { Component, Inject, OnInit } from "@angular/core";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+
+@Component({
+	selector: "showcase-demo-header",
+	templateUrl: "./header.component.html"
+})
+export class HeaderComponent implements OnInit {
+	public headerHtml: string;
+	public importSCSS: string;
+
+	public constructor(@Inject(STARK_LOGGING_SERVICE) public logger: StarkLoggingService) {}
+
+	public ngOnInit(): void {
+		this.importSCSS = `@import "~@nationalbankbelgium/stark-ui/assets/themes/app-header-theme";`;
+
+		this.headerHtml = `
+		<header class="stark-app-header">
```

Comment:
This HTML of this example is outdated, now the names of the CSS classes `stark-content-xxxx` have changed ;)